src/components/Sidebar/NavContent.jsx

```diff
@@ -107,7 +108,7 @@ function NavContent({ handleDrawerToggle }) {
                                 <Typography variant="body2">
                                   {bond.displayName}
                                   <span className="bond-pair-roi">
-                                    {`${bond.discount && trim(bond.discount * 100, 2)}%`}
+                                    <DisplayBondDiscount key={bond.index} bond={bond}></DisplayBondDiscount>
```

Comment:
Nit but this can be simplified to `<DisplayBondDiscount key={bond.index} bond={bond}/>`